README.md

```diff
@@ -27,54 +27,48 @@ If you use any [functions](https://github.com/krzkaczor/ts-essentials/blob/maste
 
 `ts-essentials` is a set of high-quality, useful TypeScript types that make writing type-safe code easier.
 
-- [Install](#Install)
```

Comment:
can you please revert the changes here?

it looks a bit annoying as you don't really change anything here

if we want to run all the files in Prettier, we can do it in the different PR

WDYT?